pkg/config/config.go

```diff
@@ -35,6 +35,7 @@ type Config struct {
 	Node          TRISAConfig         `split_words:"true"`
 	DirectorySync DirectorySyncConfig `split_words:"true"`
 	TRP           TRPConfig           `split_words:"true"`
+	Sunrise       SunriseConfig       `split_words:"true"`
```

Comment:
Will the `SunriseConfig` need to be added to the scene so that the FE may check to see if it has been enabled?

Reply:
Good point! I'll add it in this PR.